server/serializer/subscription.go

```diff
@@ -52,10 +53,21 @@ func (s *SubscriptionResponse) GetFormattedSubscription() string {
 		subscriptionEvents.WriteString(event)
 	}
 
+	var subscriptionFilters strings.Builder
+	var filtersData string
+	if len(s.FiltersData) == 0 {
+		filtersData = "N/A "
```

Comment:
```suggestion
		filtersData = "N/A"
```
Ignore if it was intentional

Reply:
yes, it was intentional

Reply:
updated the logic now